projects/igniteui-angular/migrations/common/tsUtils.ts

```diff
@@ -339,6 +362,24 @@ export const isMemberIgniteUI =
             && change.definedIn.indexOf(typeDef.name) !== -1;
     };
 
+/**
+ * Looks up a node which end property matches the specified position.
+ */
+const findNodeAtPosition = (sourceFile: tss.SourceFile, position: number): tss.Node | null => {
+    if (!sourceFile) {
+        return null;
+    }
+
+    return findInnerNode(sourceFile, position);
+}
+const findInnerNode = (node: tss.Node, position: number): tss.Node | null => {
+    if (position <= node.getEnd()) {
+        return node.forEachChild(cn => findInnerNode(cn, position)) || node;
```

Comment:
This seems like it can be improved, since the way it's set up right now, there's no way to short-circuit the recursion (might not be ideal for this case) from processing larger part of the tree than necessary.

Reply:
Alright, the docs can clear it up, it's just the `forEachChild` method is not that intuitively name, but it does exactly what we need and internally terminated the loop